.github/workflows/conform-pr.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Conform PR
+on:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
+  pull_request:
+    types:
+      # - assigned
+      # - unassigned
+      # - labeled
+      # - unlabeled
+      - opened
+      - edited
+      # - closed
+      - reopened
+      - synchronize
+      # - converted_to_draft
+      # - ready_for_review
+      # - locked
+      # - unlocked
+      # - review_requested
+      # - review_request_removed
+      # - auto_merge_enabled
+      # - auto_merge_disabled
```

Comment:
To make it easier to pick what is needed

Reply:
I'd add this comment as a comment in this file. But makes sense, yes.

Reply:
Added